Formula/autoenv.rb

```diff
@@ -16,4 +16,11 @@ def caveats; <<-EOS.undent
       source #{opt_prefix}/activate.sh
     EOS
   end
+
+  test do
+    (testpath/"test").mkdir
+    (testpath/"test/.env").write "echo it works"
+    testcmd = "yes | bash -c '. #{opt_prefix}/activate.sh; autoenv_cd test'"
```

Comment:
Oh, and `pipe_output("bash -c 'blah blah'", "y\n")` is much less understandable as far as I am concerned. I know what `yes` commonly does. I don't know what the heck `y\n` is.